it/v2/endpoints/SubmitEndOfPeriodStatementControllerTysISpec.scala

```diff
@@ -34,29 +34,6 @@ class SubmitEndOfPeriodStatementControllerTysISpec extends V2IntegrationBaseSpec
 
```

Comment:
1) Is there a special reason that the TYS updates have been given their own "TysISpec" file instead of being added to the existing ISpec file? 
2) If the Tys updates are being given their own ISpec, then I don't think we need to separate the "Test" trait (line 163) and the "TysIfsTest" trait (line 178). We should be able to put all the required test values in a generic "Test" trait

Reply:
Keeping Test and TysIfTest separate was deliberate, so that it follows the same pattern as the other TYS tests, i.e. keep it consistent/familiar.